_harp/public/pages/git.md

```diff
@@ -119,7 +119,8 @@ https://robots.thoughtbot.com/5-useful-tips-for-a-better-commit-message
   * If related to an issue provide a link to that issue
 
 Example:
-```
+
+```shell
 Clear Errors on Modal Close
 
 Clear JS errors when User closes login or Registraition modals.
```

Comment:
Git commits should follow our all lowercase convention, followed by the issue number (if applicable).

Example: "clear errors on modal close"

Reply:
👍